spec/mailers/order_notifier_spec.rb

```diff
@@ -1,22 +1,23 @@
 require "rails_helper"
+require "cgi"
 
 RSpec.describe OrderNotifier, :type => :mailer do
   include EmailSpec::Helpers
   include EmailSpec::Matchers
-  
-  let(:order) { FactoryGirl.create(:order_with_items) }
```

Comment:
Not of paramount importance but I would not remove whitespace in blocks. ;-) It is, for what I know, standard and enhances readability